test/suite/extension.test.ts

```diff
@@ -17,7 +17,7 @@ suite("Extension Test Suite", () => {
     });
 
     test("Can view static beans and mappings", async () => {
-        await vscode.commands.executeCommand("spring-boot-dashboard.focus");
+        await vscode.commands.executeCommand("spring.apps.focus");
```

Comment:
@cscherryy I don't understand why renaming would break the case. Can you help to take a look?